internal/client/project_environment.go

```diff
@@ -0,0 +1,91 @@
+package infisicalclient
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func (client Client) CreateProjectEnvironment(request CreateProjectEnvironmentRequest) (CreateProjectEnvironmentResponse, error) {
+	var body CreateProjectEnvironmentResponse
+	response, err := client.Config.HttpClient.
+		R().
+		SetResult(&body).
+		SetHeader("User-Agent", USER_AGENT).
+		SetBody(request).
+		Post("api/v1/workspace/" + request.ProjectID + "/environments")
```

Comment:
nit: i think you can use fmt.Printf() https://www.geeksforgeeks.org/fmt-printf-function-in-golang-with-examples/